protocols/v2/subprotocols/template-distribution/src/new_template.rs

```diff
@@ -69,6 +73,36 @@ pub struct CNewTemplate {
     merkle_path: CVec2,
 }
 
+impl CNewTemplate {
```

Comment:
This block should be removed.

Reply:
Delete this entire CNewTemplate impl block?